content/en/docs/collector/getting-started.md

```diff
@@ -5,65 +5,42 @@ weight: 1
 ---
 
 If you aren't familiar with the deployment models, components, and repositories
-applicable to the OpenTelemetry Collector, first review the [Data Collection
-documentation](../../concepts/data-collection).
+applicable to the OpenTelemetry Collector, first review the [Data Collection](../../concepts/data-collection)
+and [Deployment Methods](../deployment-methods) documentation.
```

Comment:
- Replace "documentation" by "pages".
- Note that the link title and path might need updating if you agree to rename & retitle the refactored page.

Reply:
fixed at 0941baf